src/components/play/PlayRoute.js

```diff
@@ -208,15 +193,11 @@ class PlayRoute extends Component {
                         AudioEngine.addUser(userBus.id, userBus.fx)
                     }
                 }
-                if (change.type === 'removed') {
```

Comment:
can we delete this?

Also is there a way to make 'removed', 'added', 'modified' something like an enum?

Reply:
Yeah, enums would definitely be better.